pageserver/src/aux_file.rs

```diff
@@ -1,11 +1,18 @@
 use bytes::{Buf, BufMut, Bytes};
 use pageserver_api::key::{Key, AUX_KEY_PREFIX, METADATA_KEY_SIZE};
+use sha2::Digest;
 use tracing::warn;
 
+fn hash256(data: &[u8]) -> [u8; 32] {
+    let mut hasher = sha2::Sha256::new();
+    hasher.update(data);
+    hasher.finalize().into()
+}
+
 /// Create a metadata key from a hash, encoded as [AUX_KEY_PREFIX, 2B directory prefix, first 13B of 128b xxhash].
```

Comment:
This refers to xxhash

Reply:
fixed in https://github.com/neondatabase/neon/pull/7644/commits/e76b2061bd75a2f68cf6750c5fe8928d985605a7